portfolio/src/main/webapp/index.html

```diff
@@ -6,7 +6,7 @@
     <link rel="stylesheet" href="style.css">
     <script src="script.js"></script>
   </head>
-  <body>
+  <body onload="getComments()">
```

Comment:
Nit: This is a small suggestion that you can maybe do in a follow-up, but you may want to remove this and instead use "window.onload = getComments;" at the end of script.js. Including it in your HTML works, but I think it's good practice to put this in script.js

Reply:
Here is a link for more clarification: https://stackoverflow.com/questions/191157/window-onload-vs-body-onload

Reply:
Thank you, I'll take a look